djangocms_page_meta/admin.py

```diff
@@ -83,21 +82,22 @@ def get_model_perms(self, request):
 admin.site.register(TitleMeta, TitleMetaAdmin)
 
 
-class UpdatedPageAdmin(PageAdmin):
+# Monkey patch the PageAdmin with a new get_form method
+_BASE_PAGEADMIN__GET_FORM = PageAdmin.get_form
+
+
+def get_form(self, request, obj=None, **kwargs):
     """
-    Remove the meta description field from the page admin
+    Patched method for PageAdmin.get_form.
 
-    It's overriden by djangocms-page-meta anyway
+    Returns a form without the base field 'meta_description', which is
+    overridden in djangocms-page-meta.
     """
-    def get_form(self, request, obj=None, **kwargs):
-        language = get_language_from_request(request, obj)
-        form = super(UpdatedPageAdmin, self).get_form(request, obj, **kwargs)
-        if obj and not obj.get_meta_description(language=language):
-            try:
-                del form.base_fields['meta_description']
-            except KeyError:
-                pass
-
-        return form
-admin.site.unregister(Page)
-admin.site.register(Page, UpdatedPageAdmin)
+    language = get_language_from_request(request, obj)
+    form = _BASE_PAGEADMIN__GET_FORM(self, request, obj, **kwargs)
+    if obj and not obj.get_meta_description(language=language):
+        form.base_fields.pop('meta_description', None)
```

Comment:
Although I personally prefer the original `try`-`except` pattern, it's challenging to test the `except` path, and that little bit of coverage difference was triggering the Codecov limits and failing this PR.  Using `pop` works and allows for 100% diff coverage.